base/broadcast.jl

```diff
@@ -1242,7 +1242,31 @@ macro __dot__(x)
     esc(__dot__(x))
 end
 
-@inline broadcasted_kwsyntax(f, args...; kwargs...) = broadcasted((args...)->f(args...; kwargs...), args...)
+"""
+    Base.Broadcast.FixKwargs{F,K} <: Function
+
+An instance `c` of `FixKwargs` is a function that takes positional
```

Comment:
Maybe mirror the definitions from `FixN`?  I often find symmetrical docs help me understand the behaviors better.

>     Fix1(f, x)
>
>  A type representing a partially-applied version of the two-argument function
>  `f`, with the first argument fixed to the value "x". In other words, `Fix1(f, x)`
> behaves similarly to `y->f(x, y)`.

Thus:

>     FixKwargs(f; kwargs...)
>
>  A type representing a partially-applied version of the keyword argument function
>  `f`, with all keyword arguments fixed to those passed. In other words, `FixKwarg(f; kws...)`
> behaves similarly to `(args...)->f(args...; kws...)`.